### DISCUSSION TITLE
メンバー一人だけ保存出来てないらしいです

### DISCUSSION THREAD

**Discussion by Estellax1220** [proshunsuke/colmsg#53] [Q&A] [answered]
もう解決しました！ありがとうございます。
消しても大丈夫です

--------------------------------------

すみません、、
いつも使っています。ありがとうございます。
しかし最近急にメンバー一人だけ保存出来てないらしいです。

thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/message/saver.rs:212:17
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

このエラーについて何かアドバイスありますか、、？

**Comment by proshunsuke** [comment, marked as answer]
解決したようで良かったです
ちなみに何が問題でどう解決しましたか？

**Reply by Estellax1220** [reply]
フォルダ内他のファイルが混ぜちゃったらしいです。。
それを削除した後、うまく保存しました。
ありがとうございました！

**Reply by proshunsuke** [reply]
手動でフォルダ内のファイルを操作する等、フォルダ内のファイル名が規則に合わなくなってしまった場合にエラーになってしまいますね
共有ありがとうございました
